pyproject.toml

```diff
@@ -65,3 +65,9 @@ include = ["quax", "tests"]
 dev = [
     "pre-commit>=4.1.0",
 ]
+test = [
```

Comment:
I am also only just realising that we have `[dependency-groups]` here! I missed that when it came in.

It's a minor point but PEP 735 is still pretty new so I think I'd prefer to keep these as `[project.optional-dependencies]`.

Reply:
Migrating.

Reply:
Done.